DockerHubfile

```diff
@@ -17,16 +17,17 @@ RUN mkdir -p application/src/main/resources/META-INF \
 RUN gradle installDist
 
 #### Actual DockerFile
-FROM openjdk:8-jre
```

Comment:
The name of this fine is inconsistent with the other `hub.Dockerfile` no?

Reply:
Good catch, thanks

Reply:
You got the `- FROM openjdk:8-jre` but missed the `+ FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-1.4`

Reply:
Doh read the comment @dwaynebailey